Tests/PromptsTest.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using TicTacToe;
+using Xunit;
+
+namespace Tests.TicTacToe {
+
+    public class PromptsTest {
+
+        public class Prompts_GreetingShould {
+
+            public Prompts _prompts;
```

Comment:
I worry about these lines of code, specifically the fact that I believe this will be redundant code every time I write a different test that uses the Prompts object.
<img width="253" alt="screen shot 2017-11-28 at 12 20 29 pm" src="https://user-images.githubusercontent.com/10368374/33336771-9040a9dc-d436-11e7-974a-6772d1a6a1f5.png">

I tried to find a way to create a global object that I can have access to on the outer Class (PromptsTest), but I will still have redundant code with the method that I found. Any suggestions?